yarn.lock

```diff
@@ -1556,6 +1707,7 @@
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/@types/hash-sum/-/hash-sum-1.0.0.tgz#838f4e8627887d42b162d05f3d96ca636c2bc504"
 
+<<<<<<< HEAD
```

Comment:
@f3ltron I think you've committed some conflicts in here 😛

Reply:
ups i will correct it